src/bentoml/__init__.py

```diff
@@ -169,15 +169,7 @@
     validators = _LazyLoader("bentoml.validators", globals(), "bentoml.validators")
     del _LazyLoader
 
-    _NEW_SDK_ATTRS = [
-        "service",
-        "runner_service",
-        "api",
-        "depends",
-        "ContentType",
-        "Dtype",
-        "Shape",
-    ]
+    _NEW_SDK_ATTRS = ["service", "runner_service", "api", "depends"]
```

Comment:
Where do we expose ContentType now? @frostming

Reply:
bentoml.validators.ContentType